frontend/src/main/scala/bloop/engine/tasks/Tasks.scala

```diff
@@ -122,7 +123,14 @@ object Tasks {
     }
 
     // For now, test execution is only sequential.
-    Task.sequence(testTasks).map { exitCodes =>
+    val runAll: List[Task[Int]] => Task[List[Int]] =
+      if (runInParallel) {
+        Task.gather
```

Comment:
didn't go with `gatherUnordered` since not sure if users would find that behavior predictable.